backends/p4tools/modules/smith/targets/generic/target.cpp

```diff
@@ -110,8 +110,20 @@ IR::P4Control *GenericCoreSmithTarget::generateIngressBlock() const {
     }
 
     IR::IndexedVector<IR::Declaration> localDecls = declarationGenerator().genLocalControlDecls();
+
```

Comment:
Remove this block?

Reply:
> Remove this block?

I'm sorry, but which block are you referring to? This `IR::IndexedVector<IR::StatOrDecl> statOrDecls` block, or this `IR::IndexedVector<IR::Declaration> localDecls` block?

Reply:
Everything

Reply:
I hope I understood it correctly, but I removed all the content in the function `GenericCoreSmithTarget::generateIngressBlock` and left only the function declaration.

```
IR::P4Control *GenericCoreSmithTarget::generateIngressBlock() const {
}
```

Reply:
Oh I meant, just remove everything you added. Just the green parts in the Github diff.